src/main/java/appeng/debug/TileChunkLoader.java

```diff
@@ -42,8 +42,8 @@
 public class TileChunkLoader extends AEBaseTile
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a constructor to the class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1258)